openrpc/src/methods/simulateTransaction.json

```diff
@@ -91,6 +91,32 @@
                     },
                     "required": [ "minResourceFee", "transactionData" ]
                 },
+                "stateChanges": {
+                    "type": "array",
+                    "description": "(optional) - On successful simulation of `InvokeHostFunction` operations, this field will be an array of `LedgerEntry`s before and after simulation occurred. Note that _at least_ one of `before` or `after` will be present: `before` and no `after` indicates a deletion event, the inverse is a creation event, and both present indicates an update event. Or just check the `type`.",
+                    "items": {
+                        "type": "object",
+                        "properties": {
+                            "^type$": {
+                                "oneOf": [1, 2, 3],
```

Comment:
checked the preview, this rendered a little strange, showing just a dropdown of 3 `any`:
<img width="619" alt="Screen Shot 2024-05-13 at 9 43 53 AM" src="https://github.com/stellar/stellar-docs/assets/10703845/469f7c30-7777-4ad8-be37-2dc602ef5be6">

in case this was not expected outcome?

Reply:
fixed!